action-check-tag-version/action.yaml

```
@@ -0,0 +1,57 @@
+name: Check tag and version
+description: 'Check the given tag agains the local repository version and main branch'
+
+inputs:
+  tag:
+    required: true
+    description: tag to check agains the local version
+  version:
+    required: true
+    description: local repository version
+  artifacts-to-promote:
+    required: true
+    description: artifacts name to promote
+
+runs:
+  using: "composite"
+  steps:
+    - name: Ensure git is installed
+      shell: bash
+      run: |
+        git --version
+    - name: Check development branch'
```

Is the single quote expected ?
```suggestion
    - name: Check development branch
```

---

nope lol typo